spec/elasticsearch_cluster_spec_helper.rb

```diff
@@ -9,45 +9,37 @@ module ElasticsearchClusterSpecHelper
 
   ELASTICSEARCH_TEST_INDEX     = "test_example_index"
   ELASTICSEARCH_TEST_LOCALHOST = ENV["ELASTICSEARCH_TEST_HOST"].presence || "127.0.0.1"
-  ELASTICSEARCH_TEST_PORT      = ENV["ELASTICSEARCH_TEST_PORT"]&.to_i || 9250
-
-  # If using Minitest, use Minitest::Hooks.
-
-  # `export ELASTICSEARCH_TEST_PORT=9200` to run tests against your local, running Elasticsearch
-
-  # If you run into issues with Webmock, make sure to allow local requests
-  # WebMock.disable_net_connect!(allow_localhost: true) # allow local requests in setup
-  # WebMock.disable_net_connect! # disallow all requests in teardown
-
-  CLUSTER_COMMANDS =
-    if ENV["SEMAPHORE_CI_ELASTICSEARCH"].present?
-      { command: "/usr/share/elasticsearch/bin/elasticsearch", es_params: "-E path.conf=/etc/elasticsearch/" }
-    else
-      {}
-    end.freeze
+  ELASTICSEARCH_TEST_PORT      = ENV["ELASTICSEARCH_TEST_PORT"]&.to_i || 9200
```

Comment:
Port 9200 is used in the `docker-compose` file and the default port for our Elasticsearch container on the buildkite agent. There isn't any reason to have this default to 9250 anymore